data/formats-data.js

```diff
@@ -6274,7 +6335,7 @@ exports.BattleFormatsData = {
 	},
 	pheromosa: {
 		randomBattleMoves: ["highjumpkick", "uturn", "icebeam", "poisonjab", "bugbuzz"],
-		randomDoubleBattleMoves: ["highjumpkick", "uturn", "icebeam", "poisonjab", "bugbuzz", "protect"],
+		randomDoubleBattleMoves: ["highjumpkick", "uturn", "icebeam", "poisonjab", "bugbuzz", "protect", "lowkick", "speedswap"],
```

Comment:
Not sure about Low Kick. Hawlucha and Mega Lopunny have High Jump Kick as its only Fighting STAB and nobody's complained about that yet. For the other HJK users in doubles, their other Fighting STAB has extra effects (Drain Punch, Mach Punch) while this is just raw damage that is outclassed by HJK all the time.